src/app/styles.ts

```diff
@@ -15,18 +15,20 @@
     You should have received a copy of the GNU General Public License
     along with this program.  If not, see <https://www.gnu.org/licenses/>.
 */
-import { makeStyles } from "@material-ui/core";
+import { Paper, styled } from '@mui/material';
 
-export const useStyles = makeStyles((theme) => ({
-  root: {
-    display: "flex",
-  },
-  viewPaper: {
-    margin: "45px 0px 0px 75px",
-    padding: "15px",
-    boxShadow: "0px 0px 0px 0px",
-  },
-  alert: {
-    marginBottom: "2rem",
-  },
-}));
+// export const useStyles = makeStyles((theme: any) => ({
```

Comment:
I would suggest to remove the commented-out lines here. Unless they're needed for some reason I think it's best to keep `main` clean from comments